server/models/orderModel.js

```diff
@@ -0,0 +1,49 @@
+import mongoose from "mongoose";
+const { Schema } = mongoose;
+
+const orderSchema = mongoose.Schema(
+  {
+    userId: {
+      type: Schema.Types.ObjectId,
+      ref: "User",
+      required: true,
+    },
+    products: [
+      {
+        productId: {
+          type: Schema.Types.ObjectId,
+          ref: "Product",
+          required: true,
+        },
+        quantity: {
+          type: Number,
+          required: true,
+          default: 1,
+        },
+      },
```

Comment:
Products should contain product price. This will ensure we save the correct price at the time of purchase.

Reply:
done